pages/task/read/routers/view.js

```diff
@@ -8,7 +8,12 @@ const { cleanModel } = require('../../../../lib/utils');
 const getContent = require('../content');
 const { getNacwoById, getEstablishment } = require('../../../common/helpers');
 
-const getRelevantActivity = activityLog => activityLog.filter(log => {
+const getRelevantActivity = activityLog => activityLog.filter((log, index) => {
+  if (get(log, 'event.meta.payload.data.extended') && index > 0) {
+    log.eventName = 'status:deadline-extension';
```

Comment:
I've overwritten the event name here (it would be a generic 'update') with a made up status so that we can reuse the ExtraMeta component with the fewest amount of changes. Not sure how you guys will feel about that?

Reply:
Fine.